kernel/test/unityIntegration/ecs/math/quaternion.test.ts

```diff
@@ -2,23 +2,23 @@ import { expect } from 'chai'
 import { Quaternion, Vector3 } from 'decentraland-ecs/src'
 
 const results = {
-  staticAngle01: '90',
-  staticAngle02: '14.8557',
+  staticAngle01: '89.99',
+  staticAngle02: '14.89',
   staticAngle03: '4.99997',
-  staticAngle04: '180',
+  staticAngle04: '179.9',
   staticEuler01: '(0.0, 0.0, 0.0, 1.0)',
   staticEuler02: '(0.5, -0.5, 0.5, 0.5)',
   staticEuler03: '(0.0, 0.9, -0.4, 0.0)',
   staticEuler04: '(0.8, 0.0, 0.6, 0.0)',
   staticEuler05: '(-0.7, 0.2, -0.2, -0.6)',
-  staticFromToRotation01: '(0.0, 0.0, 0.0, 1.0)',
-  staticFromToRotation02: '(0.0, 0.7, 0.0, 0.7)',
-  staticFromToRotation03: '(0.0, 0.0, 0.4, 0.9)',
-  staticFromToRotation04: '(0.4, 0.0, 0.0, 0.9)',
-  staticFromToRotation05: '(0.7, 0.0, -0.7, -0.3)',
-  staticFromToRotation06: '(0.3, -0.2, 0.0, 0.9)',
-  staticFromToRotation07: '(0.1, 0.4, -0.1, 0.9)',
-  staticFromToRotation08: '(0.3, 0.0, -0.3, 0.9)',
+  staticFromToRotation01: '(0.0, 180.0, 0.0)',
+  staticFromToRotation02: '(0.0, 93.9, 0.0)',
+  staticFromToRotation03: '(0.0, 0.0, 44.5)',
+  staticFromToRotation04: '(45.0, 360.0, 360.0)',
+  staticFromToRotation05: '(335.2, 276.2, 152.5)',
+  staticFromToRotation06: '(39.1, 335.6, 352.5)',
+  staticFromToRotation07: '(12.8, 45.1, 351.9)',
+  staticFromToRotation08: '(35.3, 345.0, 315.0)',
```

Comment:
I'm worried about a regression here, I trust you having this values checked. This actually deletes the tests for quaternions being generated correctly with the fromToRotation.. Want me to contribute with a commit?

Reply:
Just for the record I'll state here what we talked offline:
- Those test results were changed to euler checks because in Unity we didn't get the exact same Quaternionr, but got quaternions representing the same euler as the quaternions produced by our SDK. (I added a comment stating this but it's further below, where the test runs
- I re-tested the gimbal lock fix working fine in the original test scene that triggered this investigation
- I tested objects rotation in the world to see that nothing broke, also playing The Salmonicon that has heavy usage of rotations (the monster chase, the monster speed if you look at it with a certain angle)